### PR TITLE
ESC-1408: content changes

### PR DESCRIPTION
**Pull request by ymabdulahi**

### CODE REVIEW

File: app/uk/gov/hmrc/eusubsidycompliancefrontend/views/AddClaimDatePage.scala.html [hunk 1]
```diff
@@ -85,6 +85,9 @@
 }
 
   @formHelper(action = controllers.routes.SubsidyController.postClaimDate) {
+<legend class="govuk-fieldset__legend--xl">
```
Comment (habeeb01) [inline app/uk/gov/hmrc/eusubsidycompliancefrontend/views/AddClaimDatePage.scala.html:88]: I don't think the `legend` element is required here

File: test/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/SubsidyControllerSpec.scala [hunk 2]
```diff
@@ -230,7 +230,7 @@ class SubsidyControllerSpec
           findHintText shouldBe s"For example, 15 3 ${year}"
         }
 
-        "legend is displayed as H1 in fieldset component" in {
+        "H1 and paragraph both have legend both displayed in fieldset component" in {
```
Comment (habeeb01) [inline test/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/SubsidyControllerSpec.scala:233]: The h1 does not need to be in the fieldset

File: app/uk/gov/hmrc/eusubsidycompliancefrontend/views/AddClaimDatePage.scala.html (hunk 1 above)
Comment (ymabdulahi) [reply app/uk/gov/hmrc/eusubsidycompliancefrontend/views/AddClaimDatePage.scala.html:88]: The legend is needed here for the styling of the header

File: test/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/SubsidyControllerSpec.scala (hunk 2 above)
Comment (ymabdulahi) [reply test/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/SubsidyControllerSpec.scala:233]: can you expand on this one?

File: test/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/SubsidyControllerSpec.scala [hunk 3]
```diff
@@ -230,7 +230,7 @@ class SubsidyControllerSpec
           findHintText shouldBe s"For example, 15 3 ${year}"
         }
 
-        "legend is displayed as H1 in fieldset component" in {
+        "H1 is displayed as a legend" in {
```
Comment (habeeb01) [inline test/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/SubsidyControllerSpec.scala:233]: the h1 should not be a legend. The caption is what will be the legend

**Review by habeeb01 — APPROVED** [review]

**Pull request merged**